packages/elements/src/components/ino-segment-button/ino-segment-button.scss

```diff
@@ -54,8 +58,24 @@ $left-right-segment-border: 0 24px 24px 24px;
     @include typography.font($size: s, $weight: none);
     letter-spacing: 1.5px;
     text-align: center;
+    border-radius: $edged-border;
+
+    @include animation.border-radius(
+      ino-button-round-to-edge,
+      $round-border,
+      $edged-border,
+      0.15s
+    );
+    &:hover {
+      border-radius: $round-border;
+      @include animation.border-radius(
+        ino-segment-button-edged-to-round,
+        $edged-border,
+        $round-border,
+        0.15s
+      );
+    }
```

Comment:
You probably took that from the `ino-button` right?
I think it would be a nice idea, to extract these styles into a custom file. Maybe create a new scss file in `src/components/base/common.scss` and provide a mixin that is imported by the `ino-button` and `ino-segment-button`?